packages/foodie-backend/src/app/vendor/VendorActions.ts

```diff
@@ -0,0 +1,17 @@
+import { VendorData } from './VendorData';
+import { objectId } from '../../utils';
```

Comment:
I suggest all utilities function pushed in a util folder

Reply:
Roger, would move that into a folder